src/components/com_tjcertificate/administrator/controllers/certificates.php

```diff
@@ -34,4 +38,107 @@ public function getModel($name = 'Certificate', $prefix = 'TjCertificateModel')
 	{
 		return parent::getModel($name, $prefix, array('ignore_request' => true));
 	}
+
+	/**
+	 * Method to delete the record from frontend.
+	 *
+	 * @return  void|boolean
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function deleteCertificate()
```

Comment:
as discussed, rename it to delete()

Reply:
And how it affects the backend delete

Reply:
if we rename to delete function then it overriding the parent function and I need take care what parent function doing bcoz it is affecting the backend view if we rename it.